apps/server/src/server.config.ts

```diff
@@ -12,6 +12,7 @@ export enum NodeEnvType {
 
 export interface IServerConfig extends ICoreModuleConfig, IUserConfig, IFilesStorageClientConfig, IAccountConfig {
 	NODE_ENV: string;
+	FEATURE_COURSE_EXPORT_ENABLED: boolean;
```

Comment:
This looks like it should be in an interface, but I don't know which one either.

Reply:
Done.